webapp/dynamic-components/wpt-env-flags.html

```diff
@@ -0,0 +1,30 @@
+<!--
+Copyright 2018 The WPT Dashboard Project. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+-->
+
+<!--
+`<wpt-env-flags>` is a component for checking default enviroment wpt.fyi
+  feature flags.
+-->
+<dom-module id="wpt-env-flags">
+  <script>
+    (() => {
+      // eslint-disable-next-line no-unused-vars
+      const DEFAULTS = JSON.parse({{ .Flags }});
```

Comment:
Is `DEFAULTS` used anywhere else? If not, can we `range` over the `Flags` slice in Go template instead of doing it in JS, which seems a bit convoluted to me?

And even if we need to do it in JS, `html/template` supports directly embedding a struct or slice in a `<script>` context, and the templating engine will automatically JSON serialize the object:

```
Non-string values can be used in JavaScript contexts. If {{.}} is

struct{A,B string}{ "foo", "bar" }
in the escaped template

<script>var pair = {{.}};</script>
then the template output is

<script>var pair = {"A": "foo", "B": "bar"};</script>
```

Reply:
Done.